pcsx2/gui/AppCoreThread.cpp

```diff
@@ -510,6 +510,15 @@ void AppCoreThread::ApplySettings( const Pcsx2Config& src )
 	{
 		_parent::ApplySettings( fixup );
 	}
+
+	if (m_ExecMode >= ExecMode_Paused)
+	{
+		// Disable Vsync when frame limiter is disabled
```

Comment:
I think it's fine where it is. I kinda just copied the comment over from the old code. I'm not sure the comment is actually necessary though since it simply describes the code (which I think is self explanatory, but maybe it isn't) instead of explaining why.